### ISSUE TITLE
Invalid configuration

### ISSUE THREAD

**Issue by gundamkid** [simplethings/EntityAuditBundle#294]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | yes/no
| BC Break report? | yes/no

I tried to ignore some columns follow your documentation but i get this error: 
````
[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]   
  Unrecognized option "global_ignore_columns" under "simple_things_entity_audit"
````

this is my configuraton:
````
simple_things_entity_audit:
    global_ignore_columns:
        - created_at
        - modified_at
    service:
        username_callable: i_care_core.audit_user_provider.callable
    audited_entities:
        - MyBundle\Entity\Member
````

What wrong with me? 

Thank you!

**Comment by tolry** [comment]
@gundamkid "global_ignore_columns" was renamed to "global_ignore_properties" as it's more precise. But the old option should still be working, **except** for `master` branch, which is **not stable** at the moment.

Which version are you using?

**Comment by gundamkid** [comment]
Thank you for your reply, i'm using version 1.0.1.

**Comment by DavidBadura** [comment]
@gundamkid this was fixed in version [v1.0.2](https://github.com/simplethings/EntityAuditBundle/releases/tag/v1.0.2). Please use always the latest patch-level version before creating an issue 😉 

Thanks!
